types.d.ts

```diff
@@ -777,6 +777,16 @@ declare module "php-parser" {
     type: Identifier | Identifier[] | null;
     attrGroups: AttrGroup[];
   }
+  /**
+   * Defines a class property hook getter & setter
+   */
+  class PropertyHook extends Statement {
```

Comment:
I am not sure Statement is the best type to extends, probably better with a Function type.

This parser directly etends the Node type
https://github.com/nikic/PHP-Parser/blob/master/lib/PhpParser/Node/PropertyHook.php